### PR TITLE
[FIX] web: user menu always shown by default (in community)

### PR DESCRIPTION
**Pull request by SimonGenin**
enterprise: https://github.com/odoo/enterprise/pull/21386

### CODE REVIEW

File: addons/web/static/src/webclient/user_menu/user_menu.js [hunk 1]
```diff
@@ -52,6 +52,6 @@ UserMenu.components = { UserMenuItem };
 
 const systrayItem = {
     Component: UserMenu,
-    isDisplayed: (env) => !env.isSmall,
+    isDisplayed: () => true,
```
Comment (aab-odoo) [inline addons/web/static/src/webclient/user_menu/user_menu.js:55]: line can be removed I think

**Review by aab-odoo — APPROVED** [review]

**Comment by SimonGenin** [comment]
@aab-odoo fixed

**Comment by aab-odoo** [comment]
Thanks @SimonGenin 

robodoo r+

**Comment by aab-odoo** [comment]
robodoo r+

**Review by aab-odoo — APPROVED** [review]

**Pull request closed without merging**